CHANGES.md

```diff
@@ -23,6 +23,10 @@
   (@tmcgilchrist, #117)
 - Add a benchmark to judge the respective performance of providing a buffer vs
   letting Yojson create an internal (#134, @Leonidas-from-XIV)
+- Add an optional `suf` keyword argument was added to functions that write
+  serialized JSON, thus allowing NDJSON output. Most functions default to not
+  adding any suffix except for `to_file` (#124) and functions writing sequences
+  of values where the default is `\n` (#135, @Lenoidas-from-XIV)
 
```

Comment:
I am not sure I did it the way you wanted but take a look and judge for yourself. I just merged these entries since the previous one was never released so to avoid confusion it is now one single thing.